### DISCUSSION TITLE
The path of downloaded magic prompt models?

### DISCUSSION THREAD

**Discussion by CosmicLaca** [adieyal/sd-dynamic-prompts#472] [Q&A] [answered]
Hi, i cant find the downloaded magic prompt models, can you help to found the path of pytorch_model.bin files?

**Comment by akx** [comment]
They will be in your Hugging Face cache directory: https://huggingface.co/docs/datasets/cache

**Reply by CosmicLaca** [reply]
I mean if i use this extension in local A11, and i see in terminal that download pytorch models, but can recognize the download path later.

**Comment by kenwud** [comment]
im also looking for this. my wifi sucks and the download fails in the command prompt. but if i could download the models manually and place them into the directory manually i m sure it would work

**Comment by gurukast** [comment]
same, where in my local (windows) system does it save the models? no convenient "models" sub folder under the extension lol

**Comment by xarDeves** [comment]
for the love of god can someone answer this ?
why is this not in the docs/readme ?

**Comment by xarDeves** [comment, marked as answer]
they were indeed in the huggingface caching directory, however in local installs (on linux) the sub-directory itself is located in:
`home/.cache/huggingface/hub`
